docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -19,6 +19,12 @@ and becomes pending (for example when the cluster is highly utilized and either
 vacated by the evicted critical add-on pod or the amount of resources available on the node changed for some other reason).
 
 ## Rescheduler: guaranteed scheduling of critical add-ons
+**Rescheduler is deprecated as of Kubernetes 1.10 and will be removed in version 1.11 in
+accordance with the [deprecation policy](/docs/reference/deprecation-policy) for beta features.**
+
+**To avoid eviction of critical pods, you must
+[enable priorities in scheduler](docs/concepts/configuration/pod-priority-preemption/)
+before upgrading to Kubernetes 1.8 or higher.**
```

Comment:
Actually, this should be "Kubernetes 1.10" or higher. In 1.9 and before, they don't need to enable prioirty and preemption.

Reply:
Done. Thanks for correction.